examples/pennylane_run_quantum_natural_gradient.py

```diff
@@ -0,0 +1,452 @@
+
+Most recent implementations
+of variational quantum algorithms have used gradient-free classical optimization
+methods, such as the Nelder-Mead algorithm. However, the parameter-shift rule
+(as implemented in PennyLane) allows the user to automatically compute 
+analytic gradients of quantum circuits. This opens up the possibility to train
+quantum computing hardware using gradient descent---the same method used to train
+deep learning models. 
+Though one caveat has surfaced with gradient descent --- how do we choose the optimal
+step size for our variational quantum algorithms, to ensure successful and
+efficient optimization?
+
+The natural gradient
+^^^^^^^^^^^^^^^^^^^^
+
+In standard gradient descent, each optimization step is given by
+
+.. math:: \theta_{t+1} = \theta_t -\eta \nabla \mathcal{L}(\theta),
+
+where :math:`\mathcal{L}(\theta)` is the cost as a function of
+the parameters :math:`\theta`, and :math:`\eta` is the learning rate
+or step size. In essence, each optimization step calculates the
+steepest descent direction around the local value of :math:`\theta_t`
+in the parameter space, and updates :math:`\theta_t\rightarrow \theta_{t+1}`
+by this vector.
+
+The problem with the above approach is that each optimization step
+is strongly connected to a *Euclidean geometry* on the parameter space.
+The parametrization is not unique, and different parametrizations can distort
+distances within the optimization landscape.
+
+For example, consider the following cost function :math:`\mathcal{L}`, parametrized
+using two different coordinate systems, :math:`(\theta_0, \theta_1)`, and
+:math:`(\phi_0, \phi_1)`:
+
+|
+
+.. figure:: ../../examples/figures/quantum_natural_gradient/qng7.png
+    :align: center
+    :width: 90%
+    :target: javascript:void(0)
+
+|
+
+Performing gradient descent in the :math:`(\theta_0, \theta_1)` parameter
+space, we are updating each parameter by the same Euclidean distance,
+and not taking into the fact that the loss function might vary at a different
+rate with respect to each parameter.
+
+Instead, if we perform a change of coordinate system (or re-parametrization)
+of the cost function, we might find a parameter space where variations in :math:`\mathcal{L}`
+are more even across different parameters. This is the case with the new parametrization
+:math:`(\phi_0, \phi_1)`; the cost function is unchanged,
+but we now have a nicer geometry in which to perform gradient descent, and a more
+informative stepsize. This leads to faster convergence, and can help avoid optimization
+becoming stuck in local minima.
+
+However, what if we avoid gradient descent in the parameter space altogether?
+If we instead consider the optimization problem as a
+probability distribution of possible output values given an input
+(i.e., `maximum likelihood estimation <https://en.wikipedia.org/wiki/Likelihood_function>`_,
+a better approach is to perform the gradient descent in the *distribution space*, which is

```

Comment:
```suggestion
a better approach is to perform gradient descent in the *distribution space*, which is
```

Reply:
I think this one sounds a bit better with the 'the'